examples/ignition/torus.yaml

```diff
@@ -0,0 +1,85 @@
+---
+ignition_version: 1
+systemd:
+  units:
+    - name: etcd3.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=etcd3
+        Conflicts=etcd2.service
+        After=docker.service
+        Requires=docker.service
+        [Service]
+        ExecStartPre=/usr/bin/mkdir -p /var/lib/etcd3
+        ExecStart=/usr/bin/docker run --net=host quay.io/coreos/etcd:v3.0.0-beta.0 -name {{.etcd_name}} -advertise-client-urls http://{{.ipv4_address}}:2379 -listen-client-urls http://0.0.0.0:2379 --initial-advertise-peer-urls http://{{.ipv4_address}}:2380 --listen-peer-urls http://{{.ipv4_address}}:2380 --initial-cluster {{.etcd_initial_cluster}} -data-dir /var/lib/etcd3
```

Comment:
These flags can also be environment vars, eg, `ETCD_LISTEN_PEER_URLS=http://{{.ipv4_address}}:2380` -- whichever is cleaner